client/client.go

```diff
@@ -28,6 +28,8 @@ type Client struct {
 	failedObjectives    chan protocols.ObjectiveId
 	receivedVouchers    chan payments.Voucher
 	chainId             *big.Int
+	store               store.Store
+	vm                  *payments.VoucherManager
```

Comment:
It would be nice if the voucher wrote to the store instead of the client having to query it directly.